components/ide/theia/leeway.Dockerfile

```diff
@@ -38,7 +38,7 @@ ENV PATH=$PATH:/usr/bin/
 
 WORKDIR /theia
 
-ENV THEIA_APP_COMMIT 90b4ba0b4c32222c7ef0946ffc30c7937e203777
+ENV THEIA_APP_COMMIT 295bd05b495b604188807a516c780254aebbeff1
```

Comment:
See also: https://github.com/gitpod-io/theia-app/pull/2

Reply:
approved

Reply:
merged and commit ID updated